Sources/Preprocessors/MatchPreprocessor.swift

```diff
@@ -61,6 +61,7 @@ class MatchPreprocessor: BasePreprocessor {
         } else {
           // no cross fade in this case, fromView is always displayed during the transition.
           fvState.opacity = nil
+          tvState.opacity = nil
```

Comment:
Thanks!

It might create some unexpected behaviour for this case. The destination view will appear immediately with your change. Which might not suite everyone. 

The `forceNonFade` modifier was originally created to remove transparent effect for the source view so that the background view won't be seen during the transition. Maybe we should change it so that if it act separately to source and destination view. If the source view has this modifier, we don't fade source view, If the destination view has this modifier, we don't fade the destination view. Right now with your change, if one of the view has this modifier, both view won't fade.

Reply:
Fair point! I can compose a different PR with the behavior your outlined in your comment (`forceNonFade`on source/target view doesn't change its opacity during transition)

Reply:
Yes, that would be great!